docroot/themes/custom/twentyseventeen/sass/components/schedule/_schedule.scss

```diff
@@ -0,0 +1,134 @@
+// Schedule
+//
+// Conference Schedule Layout
+//
+// Markup: schedule.twig
+//
+// Style guide: components.schedule
+
+.program-schedule {
+	
+}
+
+.program-schedule .views-exposed-form {
+	//border: 1px solid #f00;
+	
+}
+
+
+.program-schedule .views-exposed-form .form-item.js-form-type-select.form-type-select.form-item-sort-by,
+.program-schedule .views-exposed-form .form-item.js-form-type-select.form-type-select.form-item-sort-order,
+.program-schedule .views-exposed-form .form-item.js-form-type-select.form-type-select.form-item-field-experience-level-value,
+.program-schedule .views-exposed-form .form-item.js-form-type-select.form-type-select.form-item-field-session-track-target-id,
+.program-schedule .views-exposed-form .form-actions.js-form-wrapper.form-wrapper {
```

Comment:
@charlieartist I think that we can probably remove the `.form-item.js-form-type-select.form-type-select` from each line item. The specificity is already good enough without all those classes and may make it more difficult to debug in the future.